inference-engine/ie_bridges/python/sample/speech_sample/speech_sample.py

```diff
@@ -187,7 +193,11 @@ def main():
         log.info(f'Exported GNA embedded model to file {args.export_embedded_gna_model}')
         log.info(f'GNA embedded model export done for GNA generation {args.embedded_gna_configuration}')
         return 0
-
+        
+    if args.arch:
+        if args.arch not in ("CORE", "ARCH"):
```

Comment:
combine two ifs

Reply:
Removed - handling in argparser choices as mentioned below. :)